### ISSUE TITLE
Portal Scroll doesn't work if bound to ` button if you change language (Ё on my keyboard)

### ISSUE THREAD

**Issue by raymenn** [jeffhill/Dota2#23]

**Comment by Cryszon** [comment]
Hotkeys (at least some of them) seem to be language dependent instead of being based on the physical location on the keyboard. I have scoreboard bound to `§` which is `` ` `` on the US layout and it won't work unless I use the correct language.

**Comment by wersonit** [comment]
> Hotkeys (at least some of them) seem to be language dependent instead of being based on the physical location on the keyboard. I have scoreboard bound to `§` which is `` ` `` on the US layout and it won't work unless I use the correct language.

Same for me. I have console bound to "/" and it doesn't work if I change language.

**Comment by raymenn** [comment]
> Hotkeys (at least some of them) seem to be language dependent instead of being based on the physical location on the keyboard. I have scoreboard bound to `§` which is `` ` `` on the US layout and it won't work unless I use the correct language.

Yes. Extremely frustrating when you are mashing the button to TP to base in a bad situation and your character just won't do anything and you end up dying and losing the game

**Comment by NMOTH** [comment]
Possibly related to #158

**Comment by lonkalsim** [comment]
![Безымянный](https://user-images.githubusercontent.com/104431158/171944545-158cd81b-3e48-4a61-88b5-137775223abf.png)
can close?
